src/UI/NextPageButtonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link"
+
+interface NextPageButtonLinkProps{
+  pageName: string;
+  msg: string;
+  disabled?: boolean;
```

Comment:
```suggestion
  disabled: boolean;
```

This is either true or false anyway